ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -3355,6 +3355,7 @@ peer-filter PF2
    30 match as-range 65000 result accept
 !
 router bfd
+   slow-timer 5000
    interval 900 min-rx 900 multiplier 50 default
```

Comment:
When trying to configure this the order is wrong. We are now attempting to match the order on EOS to make it easier to compare with a running config. Could you please move it?

```eos
(config-s-claus)#show sess diff
--- system:/running-config
+++ session:/claus-session-config
+router bfd
+   interval 900 min-rx 900 multiplier 50 default
+   multihop interval 300 min-rx 300 multiplier 3
+   slow-timer 5000
+   local-address 192.168.255.1
+   session stats snapshot interval 51
+   !
+   sbfd
+      local-interface Loopback0 ipv4 ipv6
+      initiator interval 500 multiplier 3
+      initiator measurement delay round-trip
+      reflector min-rx 600
+      reflector local-discriminator 155.1.3.1
smv033(config-s-claus)#
```

Reply:
yup, would be great if the final eos output would follow the order of the suggestions, so in this case:

config-router-bfd)#?
  authentication  Configure BFD authentication
  hardware        Configure hardware-specific BFD parameters
  interval        Set transmit rate in milliseconds
  ip              IP config commands
  ipv6            IPv6 config commands
  local-address   Configure BFD local IP address
  multihop        Multihop BFD session
  peer            Specify peer of BFD session
  sbfd            Seamless BFD
  session         BFD session settings
  shutdown        Set BFD admin down globally
  slow-timer      Set slow timer rate in milliseconds
  vrf             Specify VRF encapsulating peers

>> auth before hardware before interval before ip.. etc :D